src/styles/utils/functions.scss

```
@@ -0,0 +1,3 @@
+@function productWidth($card-width, $card-margin-right) {
```

Functions name should start with a verb
```suggestion
@function calcWidth($card-width, $card-margin-right) {
```